seed/analysis_pipelines/better.py

```diff
@@ -459,8 +479,11 @@ def _better_building_service_request(bsync_xml):
     }
     try:
         response = requests.request("POST", url, headers=headers, data=bsync_content)
-        data = response.json()
-        building_id = data['id']
+        if response.status_code == 201:
+            data = response.json()
+            building_id = data['id']
+        else:
+            raise
```

Comment:
Will this work? Since we aren't in an `except` I'm not sure this will work as is
```suggestion
            raise Exception(f'Received non 2xx status from BETTER: {response.status_code}: {response.content}')
```